sympy/physics/units/systems/cgs.py

```diff
@@ -66,6 +66,8 @@
 cgs_gauss.set_quantity_scale_factor(ohm, 10**5/speed_of_light**2*second/centimeter)
 cgs_gauss.set_quantity_scale_factor(farad, One/10**5*speed_of_light**2*centimeter)
 cgs_gauss.set_quantity_scale_factor(henry, 10**5/speed_of_light**2/centimeter*second**2)
+cgs_gauss.set_quantity_scale_factor(angstrom, 1*centimeter/10**8)
```

Comment:
Well, this one is not needed because it's already handled by `.set_global_relative_scale_factor( ... )`. The method `.set_quantity_scale_factor( ... )` is supposed to be used for scale factors that depend on the unit system. The scale factor for `angstrom` is the same in all unit systems.

Reply:
Just to be clear:

```python
In [1]: from sympy.physics.units.systems import SI

In [2]: from sympy.physics.units.systems.cgs import cgs_gauss

In [3]: from sympy import *; from sympy.physics.units import *

In [5]: angstrom = angstroms = Quantity("angstrom", latex_repr=r'\r{A}')

In [6]: angstrom.set_global_dimension(length)

In [10]: angstrom.set_global_relative_scale_factor(sympify(1)/10**10, meter)

In [11]: convert_to(angstrom, meter, SI)
Out[11]: meter/10000000000

In [12]: convert_to(angstrom, meter, cgs_gauss)
Out[12]: meter/10000000000

In [13]: convert_to(angstrom, centimeter, cgs_gauss)
Out[13]: centimeter/100000000

In [14]: convert_to(angstrom, centimeter, SI)
Out[14]: centimeter/100000000
```

That is, if you set the scale factor and dimensions to be global, you don't need to specify them for all unit systems.

The scale factor between `angstrom` and other usual length units (meters, centimeters, inches, feet, light years...) is always the same in all unit systems, so it has to be specified as "global".